FluentAssertions.Core/Common/CSharpAccessModifier.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace FluentAssertions.Common
+{
+    public enum CSharpAccessModifier
+    {
+        Public,
+        Private,
+        Protected,
+        Internal,
+        ProtectedInternal,
+        None
```

Comment:
Consider a better name... this means that C# cannot interperet the access modifier, but perhaps VB, etc... can